scavenger-collector/src/main/kotlin/com/navercorp/scavenger/repository/sql/AgentStateSql.kt

```diff
@@ -26,6 +26,7 @@ class AgentStateSql : SqlGeneratorSupport() {
         WHERE
             customerId = :customerId
             AND lastPolledAt < :lastPolledAt
+        LIMIT 100000
```

Comment:
isn't it little big? what about use 10000

Reply:
because the row count limit in vitess is 100000.I changed it to 10000.